CHANGELOG.md

```diff
@@ -161,6 +161,9 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 - Deprecated `Trainer.run_stage` in favor of `Trainer.{fit,validate,test,predict}` ([#11000](https://github.com/PyTorchLightning/pytorch-lightning/pull/11000))
 
 
+- Deprecated `Trainer.verbose_evaluate` in favor of `EvaluationLoop(verbose=...)` ([#10931](https://github.com/PyTorchLightning/pytorch-lightning/pull/10931))
```

Comment:
I would mention the verbose arg in Trainer.validate/test because that's more usable compared to instantiating the custom loop.

Reply:
out of curiosity, why do we have verbose on validate and test but not on fit?

Reply:
Because fit does not return `something = trainer.fit(...)`

But I don't know why that's the case